Cargo.lock

```diff
@@ -1066,9 +1066,15 @@ checksum = "6a987beff54b60ffa6d51982e1aa1146bc42f19bd26be28b0586f252fccf5317"
 
 [[package]]
 name = "linux-raw-sys"
-version = "0.3.6"
+version = "0.1.4"
```

Comment:
Weird how you can end up with multiple versions of a package when the dependencies don't line up.

Reply:
Yeah not sure how this happened tbh

Reply:
It just happens sometimes if some dependencies depend on different versions of the same package. I think it's ok so long as they don't interact.